notebooks/post_DL3_analysis.ipynb

```diff
@@ -0,0 +1,2633 @@
+{
```

Comment:
<p>I don't see the excluded region in the plot. Is there an issue? Or is it not displayed correctly on reviewnb?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/955/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Ah no, the axes limits are small in this case. I will expand it to show the excluded region.</p>